Podfile.lock

```diff
@@ -8,9 +8,9 @@ PODS:
     - Moya/Core (= 14.0.0)
   - Moya/Core (14.0.0):
     - Alamofire (~> 5.0)
-  - PureLayout (3.1.6)
+  - PureLayout (3.1.8)
   - ReactiveSwift (6.5.0)
-  - SalemoveSDK (0.29.0):
```

Comment:
Ohhhh, now I see why the retry request for device capabilities functionality seemingly wasn't included. Along with other improvements from 0.29.1 onwards..